src/generate/templates/ping-message-all.h.in

```diff
@@ -0,0 +1,46 @@
+// C++ implementation of the Blue Robotics 'Ping' binary message protocol
+
+//~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+// THIS IS AN AUTOGENERATED FILE
+// DO NOT EDIT
+//~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+
+#pragma once
+
+{% for definition in global %}
+#include "ping-message-{{definition}}.h"
+{% endfor %}
+
+namespace PingDeviceTypeNamespace {
+#ifdef QT_CORE_LIB
+    Q_NAMESPACE
+#endif
+    enum PingMessageId {
+        UNKNOWN = 0,
+        PING1D = 1,
+        PING360 = 2,
+    }
+#ifdef QT_CORE_LIB
+    Q_ENUM_NS(PingMessageId)
+#endif
+}
+
+namespace PingMessageIdNamespace {
```

Comment:
Do you really prefer using

`PingMessageIdNamespace::PingMessageId::PING1D_PROFILE`
over
`Ping1dId::PROFILE`?